frontend/constants.ts

```diff
@@ -1,10 +1,11 @@
 export const ANNOUNCEMENTS_POLL_INTERVAL = 10000;
-export const QUEUE_STATUS_POLL_INTERVAL = 30000;
+export const QUEUE_STATUS_POLL_INTERVAL = 10000;
```

Comment:
just checking, we meant to change on purpose so we can have the queue pin UI be shown immediately when the queue opens right? With the longer poll interval, you were experiencing some delay?